specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -0,0 +1,245 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SipRouting Service",
+    "description": "SipRouting Service",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2021-05-01-preview"
+  },
+  "paths": {
+    "/sip": {
+      "get": {
+        "tags": [
+          "SipConfiguration"
+        ],
+        "summary": "Gets SIP configuration for resource.",
+        "operationId": "GetSipConfiguration",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok.",
+            "schema": {
+              "$ref": "#/definitions/SipConfiguration"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get SIP configuration for resource": {
+            "$ref": "./examples/GetSipConfiguration.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "SipConfiguration"
+        ],
+        "summary": "Patches SIP configuration for resource.",
+        "operationId": "PatchSipConfiguration",
+        "consumes": [
+          "application/merge-patch+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Configuration patch.",
+            "schema": {
+              "$ref": "#/definitions/SipConfigurationPatch"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Patched configuration is returned.",
+            "schema": {
+              "$ref": "#/definitions/SipConfiguration"
+            }
+          },
+          "415": {
+            "description": "Unsupported media type of the patch."
```

Comment:
@jhendrixMSFT - Do you know if we need to mark error status with `x-ms-error-response: true"? this is not an issue for Java but want to see Go is fine with that.

Reply:
Any response code in the list to be treated as an error requires `x-ms-error-response:true`.  This should be true for all languages.  How is it working in Java?

Reply:
I added 'x-ms-error-response: true' into the error responses.

Reply:
@jhendrixMSFT actually java is not doing anything special here, it depends on whether the m4 models a http-code as an error or not, so not different from other languages. thanks for providing feedback.